plugins/base/src/main/resources/dokka/styles/style.css

```diff
@@ -550,8 +548,8 @@ h1 {
 
 
 h1.cover {
-    font-size: 60px;
-    line-height: 64px;
+    font-size: 52px;
+    line-height: 56px;
```

Comment:
Also barely noticable, makes longer names fit a bit better and doesn't grab as much attention compared to the rest of the page.